webapp/app/containers/Restore/Filters/Groups/actions.js

```diff
@@ -0,0 +1,33 @@
+//
+// Groups filter restore actions
+//
+
+import request from 'utils/request';
+import {
+  GROUPS,
+  CURRENT_GROUP,
+} from './constants';
+
+export function getCurrentGroup(currentGroup) {
+  return {
+    type: CURRENT_GROUP,
+    currentGroup,
+  };
+}
+
+export function getGroups(groups) {
+  return {
+    type: GROUPS,
```

Comment:
Maybe try to write some more accurate type name, since redux go through every reducer once an action is fired, we might triger some conflicts if not